src/UserStory/SingleStory.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { deletestory } from './Actions/deleteStory'
+
+class story extends Component {
+
+    handleClick = () => {
+        this.props.deletestory(this.props.story.id)
+        this.props.history.push('/');
+    }
+
+    render(){
+        console.log(this.props)
+        const story = this.props.story ? (
+            <div className="story">
+                <h4 className="center">{this.props.story.title}</h4>
+                <p>{this.props.story.body}</p>
+                <div className="center">
+                    <button className="btn grey" onClick = {this.handleClick}>
+                        Delete story
+                    </button>
+                </div>
+            </div>
+        ) : (
+            <div className="center">Loading story...</div>
+        )
+        
+        return (
+            <div className="container">
+                { story }
+            </div>
+        )
+    }
+}
+
+const mapStateToProps = (state, ownProps) => {
+  let id = ownProps.match.params.story_id
```

Comment:
- should declare a variable with `const` if it won't be changed
- Please refer my above comment about shortening properties extraction

Reply:
I replaced the const, however im not sure on how to shorten this, can you check my code to see if i do it correctly?